app/tests/integration/pages/experiment_timeline_and_population.py

```diff
@@ -121,19 +119,3 @@ def firefox_max_version(self, version=None):
         for item in versions:
             if item.get_attribute("value") == version:
                 item.click()
-
-    @property
-    def platform(self):
-        element = self.find_element(*self._platform_locator)
-        platforms = element.find_elements(*self._platform_option_locator)
-        for item in platforms:
-            if item.get_attribute("selected"):
-                return item.get_attribute("value")
-
-    @platform.setter
-    def platform(self, platform=None):
-        element = self.find_element(*self._platform_locator)
-        platforms = element.find_elements(*self._platform_option_locator)
-        for item in platforms:
-            if item.get_attribute("value") == platform:
-                item.click()
```

Comment:
Yeah I see your point about having to deal with the weird markup from the select box.  I guess the sanest thing to do is take these out, since they don't apply anymore, and @jrbenny35 can write new ones for the new select input as he adds the new end to end tests.

Reply:
I _could_ write new ones, but the agreement was, if you broke the tests you fix them. I'd be happy to help fix these :)

Reply:
Okay then let's leave these intact, they won't get hit if the test is skipped, and @jrbenny35 can refactor them to support the new multiselect, and that way we can reuse that logic for any future tests that need to interact with them.

Reply:
Also sorry @jrbenny35 what I meant was when we have the new end to end integration tests it will cover the case of tests that only check specific fields, not that you'd rewrite these same tests again.

Reply:
This logic is more fine-grained than needs to be tested via integration tests, in my opinion. I have front end tests covering this component.